lib/generators/aws_record/model/active_model.rb

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+module AwsRecord
+    module Generators
+      class ActiveModel
+        attr_reader :name
+  
+        def initialize(name)
+          @name = name
+        end
+  
+        # GET index
+        def self.all(klass)
+          "#{klass}.scan" # TODO No God no plz no.
```

Comment:
So let's start to think about how to resolve this. One answer is to take a single page of listed responses (already in aws-record). Another is to add "to_a" to this scan call since it's lazy loading (which is often terrifying). Ideally we can add pagination attributes to this view by default - what would that take?